packages/components/pie-button/src/index.ts

```diff
@@ -196,6 +197,23 @@ export class PieButton extends LitElement implements ButtonProps {
         this._handleClick();
     };
 
+    /**
+     * Template for the loading state
+     *
+     * @private
+     */
+    private renderSpinner (): TemplateResult {
+        const spinnerSize = this.size.includes('small') ? 's' : 'm';
```

Comment:
Why not `===`?

Reply:
the button sizes can be `'xsmall', 'small-productive', 'small-expressive',` and based on the design specs, we have to use the `s` size of the spinner for those button sizes. 

so instead of having something like -> 

```
this.size === 'xsmall' || this.size === "small-productive" |||  this.size === 'small-expressive' ? 's' : 'm';
```

i use the `inlcude` syntax as it should cover all cases.

Reply:
That's fair! I'd add a comment to remind others of this implicit behaviour.